Resources/views/default.html.twig

```diff
@@ -2,7 +2,7 @@
     https://doc.ez.no/display/DOCTEST/Customer+Attribute+Mappings to see the list of default field names. #}
 
 <script id="recommendations-template-{{ templateId }}" type="text/x-handlebars-template">
-    {% raw %}
+    {% verbatim %}
```

Comment:
This is a **BC break** because this bundle supports Symfony `2.6`. The `verbatim` tag was added in Twig `v1.21`  which is required since Symfony `v2.7.7`.

Possible solutions:

1. Skip this fix.
2. Bump version:
    - bump version of this bundle to `1.1` (creating `1.0`) branch. 
    - backport other fixes to `1.0`
    - add a requirement for Twig `~1.21|~2.0`.
    - bump a requirement for `symfony/symfony` to `^2.7.7`.

I prefer the second option.

Ping @andrerom @clash82